core/src/main/scala/sttp/tapir/EndpointIO.scala

```diff
@@ -202,7 +202,7 @@ object EndpointInput {
     }
     case class Oauth2[T](
         authorizationUrl: String,
-        tokenUrl: String,
+        tokenUrl: Option[String] = None,
```

Comment:
I think I'd remove the defaults here (from both fields). Defaults for fields which aren't at the end of argument lists aren't that nice to use

Reply:
corrected